apps/blestress/syscfg.yml

```diff
@@ -75,3 +75,6 @@ syscfg.vals:
 
     # Whether to save data to sys/config, or just keep it in RAM.
     BLE_STORE_CONFIG_PERSIST: 0
+
+    # ACL buf size must be able to contain CoC MPS plus ACL header
+    BLE_ACL_BUF_SIZE: MYNEWT_VAL_BLE_L2CAP_COC_MPS+4
```

Comment:
shouldn't it be (MYNEWT_VAL(MYNEWT_VAL_BLE_L2CAP_COC_MPS) + 4 ) ?

Reply:
This incorrectly expands to
``` c
#define MYNEWT_VAL_BLE_TRANSPORT_ACL_SIZE (MYNEWT_VAL_MSYS_1_BLOCK_SIZE-8)
```
and ignores +4. So we need to use `MYNEWT_VAL_` variant. I think it's the same as for `BLE_L2CAP_COC_MPS` definition in `nimble/host/syscfg.yml`